container/features/src/main/resources/features-sentinel.xml

```diff
@@ -109,4 +109,39 @@
         <bundle>mvn:org.opennms.features.telemetry.adapters/org.opennms.features.telemetry.adapters.netflow/${project.version}</bundle>
     </feature>
 
+    <!--Coordination-->
+    <feature name="sentinel-coordination-api" description="OpenNMS :: Features :: Distributed :: Coordination :: API"
+             version="${project.version}">
+        <bundle>mvn:org.opennms.features.distributed/coordination-api/${project.version}</bundle>
+    </feature>
+
+    <feature name="sentinel-coordination-common"
+             description="OpenNMS :: Features :: Distributed :: Coordination :: Common" version="${project.version}">
+        <feature version="${project.version}">sentinel-coordination-api</feature>
+        <bundle>mvn:org.opennms.features.distributed/coordination-common/${project.version}</bundle>
+    </feature>
+
+    <feature name="sentinel-coordination-zookeeper"
+             description="OpenNMS :: Features :: Distributed :: Coordination :: Zookeeper" version="${project.version}">
+        <feature version="${project.version}">sentinel-coordination-api</feature>
+        <feature version="${project.version}">sentinel-coordination-common</feature>
+        <feature version="${zookeeper.version}">zookeeper-dependencies</feature>
+        <bundle>mvn:org.opennms.features.distributed/coordination-zookeeper/${project.version}</bundle>
+    </feature>
+
+    <feature name="sentinel-coordination-shell"
+             description="OpenNMS :: Features :: Distributed :: Coordination :: Shell" version="${project.version}">
+        <feature version="${project.version}">sentinel-coordination-api</feature>
+        <bundle>mvn:org.opennms.features.distributed/coordination-shell/${project.version}</bundle>
+    </feature>
+
+    <feature name="zookeeper-dependencies" version="${zookeeper.version}" description="ZooKeeper Dependencies">
```

Comment:
We've updated the feature definition and the reference to use `zookeeper.version`, but where is zookeeper.version defined? This module doesn't see the one defined in `features/distributed/coordination/zookeeper/pom.xml`.

Given that we don't necessarily want to use this version of zookeeper globally (since it may conflict with what version the Kafka client libraries depend on), we may be better off just hard-coding the version to `3.4.7` here instead.